src/modules/wishlists/wishlists.service.ts

```diff
@@ -0,0 +1,72 @@
+import { Accommodation } from '#/modules/accommodations/entities/accommodations.entity';
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { Wishlist } from '#/modules/wishlists/entities/wishlist.entity';
+
+@Injectable()
+export class WishlistService {
+  constructor(
+    @InjectRepository(Wishlist)
+    private readonly wishlistRepository: Repository<Wishlist>,
+    @InjectRepository(Accommodation)
```

Comment:
Import service here via module import, don't inject the repository of another module. It is a bad practice